### PR TITLE
Addition of a new command capable of uploading local files to loco

### PR DESCRIPTION
**Pull request by gouaille**
This PR allow user to mass upload his files to Loco.
I will write tests about it but i'd love to have some feedback from you about it :).

### CODE REVIEW

File: src/Command/UploadCommand.php [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+class UploadCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('happyr:translation:upload')
+            ->setDescription('Upload your translations into your translator service.');
```
Comment (Nyholm) [inline src/Command/UploadCommand.php]: Add argument `--force` like when we drop a database. Also, add a text to the description about that we overriding Loco.

**Comment by Nyholm** [comment]
I think it looks good! Thank you.

**Comment by gouaille** [comment]
--force option has been added

**Comment by Nyholm** [comment]
Thank you for this PR!

**Pull request merged**